docs/getting_started.md

```diff
@@ -318,19 +324,42 @@ gh pr view
 gh pr merge
 ```
 
-When this pull request is merged, a workflow is started to detect changes to the system security plan and component definitions, and a new pull request is submitted. Wait for the pull request to be submitted before inspecting the changes. Mark the pull request as ready for review to allow the CI workflow to run.
+When this pull request is merged, a workflow is started to detect changes to the system security plan and component definitions, and new pull requests are submitted. Wait for the pull requests to be submitted before inspecting the changes. Mark the pull requests as ready for review to allow the CI workflow to run.
 
 ```bash
-watch gh pr list
-gh pr diff 2 --web # Use web to open a web browser.
-gh pr ready 2
+# watch for actions that are not complete (wait for nothing returned)
+gh run list --json status --json name --json number --jq '.[] | select(.status != "completed")'
+
+# Review PRs in CLI
+for PR_ID in $(gh pr list | grep "chore: automatic content update" | cut -f 1);
```

Comment:
I see what you are trying to accomplish here. The PR number usage was supposed to be more demonstrative and less scripted. I wonder if better context through comments around this would be more helpful to less technical users than fully scripting the process.

Reply:
With the commands listed in the doc, my expectation is I can run them as-is and it will just work.  Perhaps some more output that says what is happening would help?  Such as:

```bash
for PR_ID in $(gh pr list | grep "chore: automatic content update" | cut -f 1);
do
  echo "Reviewing differences in Pull Request #${PR_ID}."
  gh pr diff $PR_ID
done
```

Reply:
Yea, I think that is a fair expectation. I agree adding a bit of context about what is happening as you showed above would be really helpful.